### ISSUE TITLE
Windows support would be amazing

### ISSUE THREAD

**Issue by sidetraxaudio** [rmawatson/flutter_isolate#138]
Currently developing on desktop where the single threaded implications being encountered are horrific. The plugin wrapping is a must. You guys have done such good work, anyone with Windows experience to port this would benefit the community greatly.

**Comment by mikemoore13** [comment]
+1 it would be amazing

**Comment by Vivaldo-Roque** [comment]
I re-searched Google again to see if the flutter team had any solutions for using platform channels with isolates.

Found this article about "[Background isolate channels](https://medium.com/flutter/introducing-background-isolate-channels-7a299609cad8)".

**Comment by nmfisher** [comment]
@Vivaldo-Roque yes, this is actually being tracked in #128. I think the basic use case for this plugin (invoking other plugins from another isolate) is now deprecated because Flutter now has this capability out-of-the-box. I would advise anyone who simply needs to call plugin code from a background isolate to use official background isolate channels.

This plugin still supports some ancillary use cases (like calling dart.ui functions outside the main isolate) that aren't supported by Flutter. If you need that, you need to stick with this plugin.
